test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -9171,6 +9171,28 @@ BOOST_AUTO_TEST_CASE(return_external_function_type)
 	);
 }
 
+BOOST_AUTO_TEST_CASE(unitialised_function_type)
+{
+	char const* sourceCode = R"(
+		contract C {
+			function f() public returns (bool) {
+				function () internal _internal;
+				_internal();
+				return true;
+			}
+			function g() public returns (bool) {
+				function () external _external;
+				_external();
```

Comment:
I think this actually should be a success as it calls the zero address which has no code. Need to adjust the test in that case.

Reply:
It fails correctly, because calling a contract that does not have code reverts.